velox/expression/CastExpr.cpp

```diff
@@ -838,6 +858,35 @@ VectorPtr CastExpr::applyTimestampToVarcharCast(
   return result;
 }
 
+template <typename TInput>
+VectorPtr CastExpr::applyIntToBinaryCast(
+    const SelectivityVector& rows,
+    exec::EvalCtx& context,
+    const BaseVector& input) {
+  VectorPtr result = BaseVector::create<FlatVector<StringView>>(
+      VARBINARY(), rows.end(), context.pool());
+  auto flatResult = result->asFlatVector<StringView>();
```

Comment:
`BaseVector::create<FlatVector<StringView>>` returns `FlatVectorPtr<StringView>`. If you want VectorPtr, you can just call `BaseVector::create`